suite-common/formatters/src/formatters/prepareCryptoAmountFormatter.ts

```diff
@@ -116,10 +116,18 @@ export const prepareCryptoAmountFormatter = (config: FormatterConfig) =>
                 ? truncateDecimals(convertedValue, maxDisplayedDecimals, isEllipsisAppended)
                 : convertedValue;
 
+            const withoutLeadingZeros = truncatedValue
+                .replace(/^0+([0-9]\.)/, '$1')
+                .replace(/^0+([0-9])$/, '$1');
```

Comment:
This seems redundant; if I understand correctly, all we need is to remove all zeroes at beginning but leave at least one any number after them (to ensure 0 or 000 works).
So I'd simplify to this, testss pass :heavy_check_mark: 
```suggestion
            const withoutLeadingZeros = truncatedValue.replace(/^0+([0-9])/, '$1');
```

Reply:
Updated.